logger.js

```diff
@@ -0,0 +1,247 @@
+// Logger Utility with Linter Issues
+
+// Logger Levels
+const levels = {
+    info: "INFO",
+    warn: "WARN",
+    error: "ERROR",
+    debug: "DEBUG"
+}
+
+// Unused global variables
+var currentLogLevel = "info"
```

Comment:
<div>


<div id="suggestion">
<div id="issue"><b>Consider removing unused variable declarations</b></div>
<div id="fix">

Several unused variables detected in the codebase. Consider removing `currentLogLevel` if not needed, or implement its usage in the logging logic. Similar unused variables found throughout the file that should be addressed.
</div>
<details>
<summary>
<b>Code suggestion</b>
</summary>
<div id="code">

```
 -var currentLogLevel = "info"

```
</div>
</details>
</div>


<small><i>Code Review Run <a href=https://github.com/aakashgusain1990/BitoPythonTest/pull/20#issuecomment-2440426565>#7b8f80</a></i></small>
</div>

Is this a valid issue, or was it incorrectly flagged by the Agent?
- [ ] it was incorrectly flagged